tests/Schema/ComparatorTest.php

```diff
@@ -1130,34 +1091,17 @@ public function testComparesNamespaces(): void
         self::assertEquals($expected, $this->comparator->compareSchemas($fromSchema, $toSchema));
     }
 
-    public function testCompareGuidColumns(): void
-    {
-        $column1 = new Column('foo', Type::getType('guid'), ['comment' => 'GUID 1']);
-        $column2 = new Column(
-            'foo',
-            Type::getType('guid'),
-            ['notnull' => false, 'length' => 36, 'fixed' => true, 'default' => 'NEWID()', 'comment' => 'GUID 2.'],
-        );
-
-        self::assertEquals(['notnull', 'default', 'comment'], $this->comparator->diffColumn($column1, $column2));
-        self::assertEquals(['notnull', 'default', 'comment'], $this->comparator->diffColumn($column2, $column1));
-    }
```

Comment:
The comparator no longer needs to be aware of platform-specific type implementation details.